package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "blog.jonasneubert.com",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build-tw": "tailwindcss -w -i ./assets/css/main.css -o ./assets/css/style.css",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "yarn build-tw & hugo server",
+    "build": "tailwindcss  -i ./assets/css/main.css -o ./assets/css/style.css && hugo --minify"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/jonemo/blog.jonasneubert.com.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
Please drop all the metadata from package.json. Only keep what is technically necessary, see also: https://github.com/jonemo/jonasneubert.com/blob/master/package.json License, homepage, keywords, ... are only needed for published packages.

Reply:
sure .